src/Data/NormalKey.purs

```diff
@@ -101,6 +97,8 @@ instance ordNormalKey :: Ord NormalKey where
   compare (NormalKey "Shift") _ = LT
   compare _ (NormalKey "Shift") = GT
   compare (NormalKey "Enter") (NormalKey "Enter") = EQ
+  compare _ (NormalKey "Enter") = GT
+  compare (NormalKey "Enter") _ = LT
   compare (NormalKey x) (NormalKey y) = compare x y
```

Comment:
That's not an injective function, because every key other than `Control, Meta, Alt, Shift, Enter` is identified in the codomain. What the above function would do is say that `NormalKey "x" == NormalKey "y"`, for instance...

I think you'll want something that takes every `NormalKey` to a *distinct* integer.